doc/classes/CanvasItem.xml

```diff
@@ -67,7 +67,7 @@
 			<argument index="1" name="radius" type="float" />
 			<argument index="2" name="color" type="Color" />
 			<description>
-				Draws a colored circle.
+				Draws a colored, non-filled circle. See also [method draw_arc], [method draw_polyline] and [method draw_polygon].
```

Comment:
"non-filled circle" is technically redundant, but I feel it's better to be explicit here :slightly_smiling_face: